code/modules/surgery/bodyparts/_bodyparts.dm

```diff
@@ -75,13 +80,17 @@
 	var/stamina_dam = 0
 	///The maximum stamina damage a bodypart can take
 	var/max_stamina_damage = 0
-	///The maximum "physical" damage a bodypart can take. Set by children
+	///The maximum brute OR burn damage a bodypart can take. Once we hit this cap, no more damage of either type!
 	var/max_damage = 0
+
+	// Limb cremation tracking variable
 	///Gradually increases while burning when at full damage, destroys the limb when at 100
 	var/cremation_progress = 0
-	///Subtracted to brute damage taken
+
+	// Damage reduction variables for damage handled on the limb level. Handled after worn armor.
+	///Amount removed from brute damage inflicted on the limb.
```

Comment:
both of these should mention subtrwction as its a unique mechanic to lilbs unlike other damage reduction cofe

Reply:
that's a fair point. It's inaccurate to just say 'removed'. Removed _how_?